app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -121,7 +121,7 @@ def exit_status
       def update_host_status
         host = Host.find(input[:host][:id])
         status = host.execution_status_object || host.build_execution_status_object
-        status.status = exit_status.zero? ? HostStatus::ExecutionStatus::OK : HostStatus::ExecutionStatus::ERROR
+        status.status = exit_status.to_i.zero? ? HostStatus::ExecutionStatus::OK : HostStatus::ExecutionStatus::ERROR
```

Comment:
This wouldn't update the execution status correctly
```
pry(#<Actions::RemoteExecution::RunHostJob>)> exit_status
=> "EXCEPTION"

pry(#<Actions::RemoteExecution::RunHostJob>)> exit_status.to_i
=> 0

Reply:
Good catch: updated